WeakAurasOptions/ConditionOptions.lua

```diff
@@ -163,14 +163,14 @@ local function filterUsedProperties(indexToProperty, allDisplays, usedProperties
 end
 
 local function addControlsForChange(args, order, data, conditions, i, j, allProperties, usedProperties)
-  local thenText = (j == 1) and L["Then "] or "and ";
+  local thenText = (j == 1) and L["Then "] or L["And "];
 
   local display = isSubset(data, conditions[i].changes[j]) and allProperties.displayWithCopy or allProperties.display;
   local valuesForProperty = filterUsedProperties(allProperties.indexToProperty, display, usedProperties, conditions[i].changes[j].property);
 
   args["condition" .. i .. "property" .. j] = {
     type = "select",
-    name = thenText .. blueIfSubset(data, conditions[i].changes[j]) .. L["Property"],
```

Comment:
The reason there's the "Property" string, is so that it can be colored blue if the property is only in some auras.